metal/datasource_metal_connection.go

```diff
@@ -0,0 +1,141 @@
+package metal
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/packethost/packngo"
+)
+
+func connectionPortSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
```

Comment:
Should we get in the habit of including the `Description` field, for #13?

Reply:
yes, probably good idea. Just it means writing it twice until we are able to finish #13